applications_tests/gls_sharp_navier_stokes_3d/pp_contact_test.prm

```diff
@@ -87,10 +88,10 @@ subsection particles
 	set refine mesh inside radius factor = 0
 	set refine mesh outside radius factor = 2
 	set initial refinement = 3
-	set fluid density =0.001
 	set integrate motion = true
 	set alpha =1
 	set assemble Navier-Stokes inside particles = false
+	set lubrication range max=0
```

Comment:
Why not just put the lubrication force to false instead?
I would also take the opportunity of removing the parameters for which you have the same default value.

Reply:
good point !